Source/CarthageKit/Dependency.swift

```diff
@@ -4,14 +4,32 @@ import Tentacle
 
 /// Uniquely identifies a project that can be used as a dependency.
 public enum Dependency {
+	/// Uniquely identifies a Binary Spec's resolved URL and its description
+	public struct BinaryURL: CustomStringConvertible {
```

Comment:
Let's make this a top-level type.

Reply:
Will address in upcoming commit, wasn't sure if it was useful outside Dependency.